x-pack/plugins/security/public/components/breadcrumb.tsx

```diff
@@ -10,6 +10,7 @@ import type { FunctionComponent } from 'react';
 import React, { createContext, useContext, useEffect, useRef } from 'react';
 
 import { useKibana } from '../../../../../src/plugins/kibana_react/public';
+import type { ChromeStart } from '../../../../../src/core/public';
```

Comment:
nit: type imports from core should use the import alias

```suggestion
import type { ChromeStart } from 'src/core/public';
```